traveltimepy/sdk.py

```diff
@@ -48,10 +48,19 @@
 
 class TravelTimeSdk:
 
-    def __init__(self, app_id: str, api_key: str, limit_per_host: int = 2, host: str = 'api.traveltimeapp.com') -> None:
+    def __init__(
+        self,
+        app_id: str,
+        api_key: str,
+        limit_per_host: int = 2,
+        rate_limit: int = 30,
```

Comment:
perhaps 60 should be default to match plans we are selling?

Reply:
sure, changed to 60